contributed_definitions/nyaml/NXmpes.yaml

```diff
@@ -5,19 +5,24 @@
 
 doc: This is the most general application definition for multidimensional photoelectron spectroscopy.
 category: application
-NXmpes:
+    NXmpes:
```

Comment:
This has been indented accidentally I guess.

Reply:
Yes. Thanks for spotting it.